app/models/hyrax/work.rb

```diff
@@ -14,6 +14,8 @@ class Work < Hyrax::Resource
     attribute :on_behalf_of,             Valkyrie::Types::String
     attribute :proxy_depositor,          Valkyrie::Types::String
     attribute :state,                    Valkyrie::Types::URI.default(Hyrax::ResourceStatus::ACTIVE)
+    attribute :representative_id,        Valkyrie::Types::ID
+    attribute :thumbnail_id,             Valkyrie::Types::ID
```

Comment:
Required for processing in [FileSetActor #attach_to_valkyrie_work](https://github.com/samvera/hyrax/blob/master/app/actors/hyrax/actors/file_set_actor.rb#L92-L93).

Reply:
there was some discussion about other ways of doing this in slack, which seemed to land on sticking with this approach.